docs/tutorials/migrate-from-neo4j.md

```diff
@@ -0,0 +1,497 @@
+```
+
+Then,  create nodes with a certain label and properties. As an example, let's
+look at the property `_id`. To add the property to the node, define its name in
+Memgraph and assigned the value of a specific column in the CSV file. 
+
+```
+LOAD CSV FROM "/usr/lib/memgraph/shipping.csv" WITH HEADER AS row
+WITH row WHERE row._labels = ':Employee'
+CREATE (e:Employee {nodeID: row._id})
+```
+
+So `nodeID: row._id` part of the query instructs Memgraph to create a property
+named `nodeID` and assign it the value paired with key `_id`. First created
+node will be assigned the value from the first data row, second node from the
+second data row, etc.
+
+**After matching up the keys and values for all properties, the finished query looks like this:**
+
+```
+LOAD CSV FROM "/usr/lib/memgraph/shipping.csv" WITH HEADER AS row
+WITH row WHERE row._labels = ':Employee'
+CREATE (e:Employee {nodeID: row._id, employeeID: row.employeeID, firstName: row.firstName, lastName: row.lastName, title: row.title});
+
+MATCH (e:Employee)
+RETURN e; 
+```
+
+The second query will show all 9 created nodes. 
+
+Copy the query in the **Cypher Editor** and **Run Query**.
+
+<img src={require('../data/tutorials/migrate-from-neo4j/employees.png').default} className={"imgBorder"}/>
+
+#### Order nodes
+
+Relevant properties for the `Order` nodes are `_id`, `orderID` and `shipName`.
+
+**To create `Order` nodes run the following query:**
+
+```
+LOAD CSV FROM "/usr/lib/memgraph/shipping.csv" WITH HEADER AS row
+WITH row WHERE row._labels = ':Order'
+CREATE (o:Order {nodeID: row._id, orderID: row.orderID, shipName: row.shipName});
+
+MATCH (o:Order)
+RETURN o; 
+```
+
+The second query will show all 830 created nodes:
+
+<img src={require('../data/tutorials/migrate-from-neo4j/orders.png').default} className={"imgBorder"}/>
+
+#### Product nodes
+
+Relevant properties for the `Product` nodes are `_id`, `productID`, `productName`
+and `unitPrice`.
+
+As the parser parses all the values as strings, and the `unitPrice` are numbers,
+they need to be converted to appropriate data type.
+
+**To create `Product` nodes run the following query:**
+
+```
+LOAD CSV FROM "/usr/lib/memgraph/shipping.csv" WITH HEADER AS row
+WITH row WHERE row._labels = ':Product'
+CREATE (p:Product {nodeID: row._id, productID: row.productID, productName: row.productName, unitPrice: ToFloat(row.unitPrice)});
+
+MATCH (p:Product)
+RETURN p; 
+```
+
+The second query will show all 77 created nodes:
+
+<img src={require('../data/tutorials/migrate-from-neo4j/product.png').default} className={"imgBorder"}/>
+
+### 4. Graph improvements
+
+At this point it would be nice to improve the look of the nodes visually. At the
+moment, nodes in the graph are represented with their labels, but it would be
+more useful if their name attribute was written. 
+
+To adjust the look of the graph with using Graph Style Language, open the the
+Graph Style Editor. On line 22 (or in that vicinity) you will find this block of
```

Comment:
I would rephrase this sentence something along the like "look for the following code" so that line number is not mentioned.